### DISCUSSION TITLE
No available formula or cask with the name "graphviz".

### DISCUSSION THREAD

**Discussion by Jitheshxavier** [Homebrew/discussions#1409] [Casks]
Command "brew install graphviz" gives below error. Someone please help me on this.
Error: No similarly named formulae found.
Error: No available formula or cask with the name "graphviz".
It was migrated from homebrew/cask to homebrew/core.

**Comment by SMillerDev** [comment]
What do `brew doctor` and `brew config` say?

**Reply by Jitheshxavier** [reply]
`Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Suspicious https://github.com/Homebrew/homebrew-core git origin remote found.
The current git origin is:
  https://github.com/Homebrew/brew

With a non-standard origin, Homebrew won't update properly.
You can solve this by setting the origin remote:
  git -C "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core" remote set-url origin https://github.com/Homebrew/homebrew-core
`

**Reply by carlocab** [reply]
You forgot the output of `brew config`. Did you try the command suggested by `brew doctor`?
```
git -C "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core" remote set-url origin https://github.com/Homebrew/homebrew-core
```
